infra/src/main/kotlin/fi/fta/geoviite/infra/ratko/RatkoController.kt

```diff
@@ -61,4 +63,20 @@ class RatkoController(
         logger.apiCall("ratkoIsOnline")
         return ratkoStatusService.getRatkoOnlineStatus()
     }
+
+    @PreAuthorize(AUTH_ALL_WRITE)
+    @PostMapping("/fetch-operating-points-from-ratko")
```

Comment:
Voisko tää joko olla `GET` tai olla nimetty jotenkin muuten jos tää tyyliin aloittaa haun taustalle pyörimään? Meillä on muuallakin näitä `POST`-metodillisia `getJotain`-hakuja, ja ne ovat ennenkin aiheuttaneet sekaannusta (viimeksi oikkasäädön yhteydessä)